### ISSUE TITLE
Feature Request: disable dice icon globally or on a per-dice basis.

### ISSUE THREAD

**Issue by WychWitch** [valentine195/obsidian-dice-roller#54]
Hello! I was messing around with the dice roller and experimented with using a zero width character between two dice to seamlessly combine them (since putting them literally next to each other causes neither of them to work) and it works, however the dice icon from the first dice gets in the way.

Example:

![image](https://user-images.githubusercontent.com/1291820/142450185-95131691-5607-47d8-a1b8-9c3fdd37e05d.png)
![image](https://user-images.githubusercontent.com/1291820/142450227-93ef8f59-b1b9-4d9b-a2f9-3ebb25666ed3.png)

While disabling it globally would be good, also being able to add an option to use on a dice-by-dice basis (maybe `|nodice` similar to `|link`?) 

I'm sorry for all these feature requests! I have one final one that's slightly related but I'll make a separate issue for it ^^

**Comment by WychWitch** [comment]
Thank you so much!! 🎊 for this and #55  !! Now I can properly smash my names for my name gen together hahaha

**Comment by WychWitch** [comment]
OH I'M SORRY I think this introduced an issue where there's always a space just after the result, which normally would be fine except for cases where I want to use a Zero Width Character.

It also displays this way in the table itself, but when actually rolling this extra space is removed so it doesn't cause issues.

Example:
![image](https://user-images.githubusercontent.com/1291820/142486874-76f17a97-3a97-4277-b565-35f7686d209a.png)

Example Code:

```
## Main Table
| dice: 1d2 | Names                                                                   |
| --------- | ----------------------------------------------------------------------- |
| 1         | `dice: [[Name Dice Tables#^names]]`&#8203`dice: [[Name Dice Tables#^femme]]` |
| 2         | `dice: [[Name Dice Tables#^names]]`&#8203`dice: [[Name Dice Tables#^masc]]`  |
^main

`dice: [[Name Dice Tables#^names]]`&#8203`dice: [[Name Dice Tables#^masc]]`

`dice: [[Name Dice Tables#^main]]`
```
If there's anything else I can provide let me know!

**Comment by valentine195** [comment]
This is fixed in 7.4.2

**Comment by WychWitch** [comment]
Thank you!!! I cant tell you how much this helps for utilizing tables as random generators 😄
